### DISCUSSION TITLE
How to use cursorcolumn

### DISCUSSION THREAD

**Discussion by OpenFoam-User** [helix-editor/helix#4348] [Q&A] [answered]
I see that the cursorcolumn feature is added in this [PR](https://github.com/helix-editor/helix/pull/4084).

However, I have no idea how to use it. I tried the following:

```
:set cursorcolumn
```

But I get the error:

```
Bad arguments. Usage: `:set key field`
```

Could you please explain how one should use it?

Thank you

**Comment by sudormrfbin** [comment, marked as answer]
From the [`documentation entry`][1] in the PR diff, `cursorcolumn` is a bool value. So `:set cursorcolumn true` should work.

[1]: https://github.com/helix-editor/helix/pull/4084/files#diff-90493a3122e39c4af3ab5f7981bfb82ac755b79174a2fb13e801bec7d416adf0R48
